apps/dcos_dns/test/dcos_dns_mesos_tests.erl

```diff
@@ -356,6 +356,9 @@ pod_tasks_setup() ->
     setup(pod_tasks_setup).
 
 setup(SetupFun) ->
+    prometheus:start(),
```

Comment:
To make the tests work I added this here, and also changed the metrics declaration from `new` to `declare` as the latter does not require the metrics not to exist before.

Do you see a better way to solve for the fact that the tests exercise the function that calls `Prometheus`?

Reply:
Another weird observation is that locally the tests pass but now on CI, tests for the other metrics fail.